2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,54 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  let newArr = [];
+  cities.forEach((city) => {
```

Comment:
👍  for using forEach and lambdas!

Formatting is quite difficult to read through. Multiline strings looks strange. What about:

```javascript
    newArr.push(`The temperature in city is ${temperatureService(city)} degrees`);
```